go/vt/sqlparser/ast.go

```diff
@@ -4605,11 +4609,11 @@ func (node TableName) IsEmpty() bool {
 
 // ToViewName returns a TableName acceptable for use as a VIEW. VIEW names are
 // always lowercase, so ToViewName lowercasese the name. Databases are case-sensitive
-// so Qualifier is left untouched.
+// so DbQualifier is left untouched.
 func (node TableName) ToViewName() TableName {
 	return TableName{
-		Qualifier: node.Qualifier,
-		Name:      NewTableIdent(strings.ToLower(node.Name.v)),
+		DbQualifier: node.DbQualifier,
```

Comment:
From looking at the other PRs, we're using this to refer to either a schema or database, and the name change implies this is only for databases? This is definitely confusing, unless I misread the other PRs and this _is_ only for databases, which I could have.

Reply:
There's a compatibility issue with gms here ... in the `dbname.tablename` format, we really want the `dbname` element to be called the `DBQualifier`. It's only in the `dbname.schemaname.tablename` format that schema comes up. That's why the naming is how it is. Might take another pass as this gets further along.